packages/overlay/test/focus-trap.test.js

```diff
@@ -94,6 +95,7 @@ describe('focus-trap', () => {
       parent = fixtureSync('<overlay-components-wrapper></overlay-components-wrapper>');
       await nextRender();
       overlay = parent.$.defaultOverlay;
+      overlay.requestContentUpdate();
```

Comment:
What is the purpose of this `requestContentUpdate`? Doesn't the overlay take care of rendering its content once it is opened?